WordPress/WordPressTest/AccountServiceTests.swift

```diff
@@ -2,14 +2,12 @@ import UIKit
 import XCTest
 @testable import WordPress
 
-class AccountServiceTests: XCTestCase {
-    var contextManager: ContextManagerMock!
+class AccountServiceTests: CoreDataTestCase {
     var accountService: AccountService!
 
     override func setUp() {
         super.setUp()
 
-        contextManager = ContextManagerMock()
         contextManager.setUpAsSharedInstance()
```

Comment:
Correct me if I'm wrong: Once #18579 lands, we can come back and further DRY this and other tests by using the new `useAsSharedInstance(untilTestFinished:)` method here and removing the `contextManager.tearDown()` call below. Right?

Reply:
Yes, this line [was changed](https://github.com/wordpress-mobile/WordPress-iOS/pull/18579/files#diff-58a0cd66170c21c6183ad4c4347ac79df6238204d98daae8b014fb29d1ecd397) in https://github.com/wordpress-mobile/WordPress-iOS/pull/18579

Reply:
🎉